packages/data-layer/src/data.types.ts

```diff
@@ -416,3 +414,33 @@ export type Collection = {
   description: string;
   applicationRefs: string[];
 };
+
+export type Application = {
```

Comment:
is there already an Application type in here somewhere? or maybe the Allo team added one in their PR too, I think they called it `ProjectApplication`

Reply:
The ProjectApplication is a slightly different type - I would rather keep the original Application from Explorer than try to merge those at the moment.